firebase/functions/src/config.ts

```diff
@@ -11,16 +11,22 @@ sgMail.setApiKey(sgMailApiKey);
 const REGION_EUROPE_WEST_1 = 'europe-west1';
 const MAX_RESULTS = 30;
 const MAPS_ENABLED = true;
-const MINIMUM_NOTIFICATION_DELAY = 20;
+const MINIMUM_NOTIFICATION_DELAY_INITIAL_ENGAGEMENT_WITH_OPEN_ENTRIES_MINUTES = 20;
 const SEND_EMAILS = sgMailApiKey !== null;
 const sendingMailsDisabledLogMessage = 'Sending emails is currently disabled.';
 const EMAIL_NOTIFICATION_AUDIENCE_SIZE_SANITY_CHECK = 35_000;
+const MINIMUM_NOTIFICATION_DELAY_UNTIL_FURTHER_ENGAGEMENT_WITH_OPEN_ENTRIES_DAYS = 3;
```

Comment:
value and naming to be discussed

Reply:
maybe we could shorten the constant names a bit
```suggestion
// The minimum notification delay until we send the next email in days
// this only happens when the user did not resolve his request
const MINIMUM_FOLLOWUP_DELAY_DAYS = 3;
```